lib/processors/stringReplacer.js

```diff
@@ -14,8 +14,9 @@ const replaceStream = require("replacestream");
  */
 module.exports = function({resources, options}) {
 	return Promise.all(resources.map((resource) => {
-		const stream = resource.getStream()
-			.pipe(replaceStream(options.pattern, options.replacement));
+		const stream = resource.getStream();
+		stream.setEncoding("utf8");
+		stream.pipe(replaceStream(options.pattern, options.replacement));
 
 		resource.setStream(stream);
```

Comment:
as far as I can see, stream.pipe returns another pipe (the destination of the pipe operation). In the original code, that return value was set as stream in the resource and I assume the same has to be done here.

Reply:
Totally makes sense. Thanks!